thunder/core/prims.py

```diff
@@ -634,6 +634,7 @@ def unpack_trivial_impl(x: Any, /, *, name: str | None = None) -> Any:
 
 
 def unpack_trivial_meta(x: Any, /, *, name: str | None = None) -> Any:
+    utils.check(name is not None, lambda: "Expected name argmument to not be None")
```

Comment:
The type hint should be updated as well and the default value removed `name: str | None = None` -> `name: str`